main.cpp

```diff
@@ -55,14 +56,14 @@ video: Chapter 2 - Part 3
  */
 
 //2)
-void variableDeclarations()
+void variableDeclarations(float age, float time, float money, int a = 1, int b = 2, int c =3, bool d = false, bool e = true, bool f = false, char man = 'h', char ted = 't', char fred = 'g')
+
 {
     //example:
     int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
     
 
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(age,time,money,a,b,c,d,e,f,man,ted,fred,number); //passing each variable declared to the ignoreUnused() function
 }

```

Comment:
You're supposed to declare variables INSIDE the function, not in the function parameter list
```
2) for each primitive type, write out 3 variable declarations inside the variableDeclaration() function on line 59.
    a) give each variable declaration an initial value
```
Also, add space after `,` so the code is more readable.
- [ ] revise